DuggaSys/profileServices/changeProfileValues.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+date_default_timezone_set("Europe/Stockholm");
+
+include_once "../Shared/basic.php";
+include_once "../Shared/sessions.php";
```

Comment:
The includes have the wrong paths to the files. This will look inside the DuggaSys after a folder called Shared.